Mathlib/Analysis/NormedSpace/Multilinear/Basic.lean

```diff
@@ -462,27 +500,27 @@ theorem le_of_op_nnnorm_le {C : ℝ≥0} (h : ‖f‖₊ ≤ C) : ‖f m‖₊ 
 theorem op_nnnorm_le_iff {C : ℝ≥0} : ‖f‖₊ ≤ C ↔ ∀ m, ‖f m‖₊ ≤ C * ∏ i, ‖m i‖₊ := by
   simp only [← NNReal.coe_le_coe]; simp [op_norm_le_iff _ C.coe_nonneg, NNReal.coe_prod]
 
-theorem isLeast_op_nnnorm : IsLeast {C : ℝ≥0 | ∀ m, ‖f m‖₊ ≤ C * ∏ i, ‖m i‖₊} ‖f‖₊ := by
+theorem isLeast_op_nnnorm :
+    IsLeast {C : ℝ≥0 | ∀ m, ‖f m‖₊ ≤ C * ∏ i, ‖m i‖₊} ‖f‖₊ := by
   simpa only [← op_nnnorm_le_iff] using isLeast_Ici
 
-theorem op_nnnorm_prod (f : ContinuousMultilinearMap 𝕜 E G) (g : ContinuousMultilinearMap 𝕜 E G') :
-    ‖f.prod g‖₊ = max ‖f‖₊ ‖g‖₊ :=
+theorem op_nnnorm_prod (f : ContinuousMultilinearMap 𝕜 E G)
+    (g : ContinuousMultilinearMap 𝕜 E G') : ‖f.prod g‖₊ = max ‖f‖₊ ‖g‖₊ :=
   eq_of_forall_ge_iff fun _ ↦ by
     simp only [op_nnnorm_le_iff, prod_apply, Prod.nnnorm_def', max_le_iff, forall_and]
 
-theorem op_norm_prod (f : ContinuousMultilinearMap 𝕜 E G) (g : ContinuousMultilinearMap 𝕜 E G') :
-    ‖f.prod g‖ = max ‖f‖ ‖g‖ :=
+theorem op_norm_prod (f : ContinuousMultilinearMap 𝕜 E G)
+    (g : ContinuousMultilinearMap 𝕜 E G') : ‖f.prod g‖ = max ‖f‖ ‖g‖ :=
```

Comment:
Is there a reason you re-wrapped these lines?

Reply:
A `DecidableEq` hypothesis that came and went. Is there some guide style about this ? I'm honestly not sure how to arrange the lines of statements, except for the fact that I should try to keep both members of an equality (or `iff`) on the same line if they fit.

Reply:
In this particular case for example, I like my version better because the context is introduced on a line, then we have the statement of the theorem on a separate line. So I find it easier to read. 
However, I don't really care that much how statements are written, so if there's a different mathlib convention I'll adhere to it.

Reply:
I think you're reading this diff backwards; it was the *original* version that had the statement on a separate line to the contet (which I agree is a good thing)

Reply:
Perhaps what matters more here though is that you should try to avoid mixing style changes in the same PR as a refactor, as each gets in the way of reviewing the other.

Reply:
I've gone ahead and reverted these changes; it makes this PR shorter, and it's easy to recover them in a future PR with `git revert` if you want them back.

Reply:
I think I went about this wrong, let me try to start afresh and make a new commit with a minimal diff.

Also, I was not sure whether I should label this as a "refactor" or "feat". I chose "feat" because the definition of "refactor" I found said that refactoring is improving code without creating new functionality, and here the code barely changes but a (very small) new functionality appears. But maybe I should have called it a "refactor" ? I'm not very comfortable with the terminology yet.

Reply:
I'm not sure that's necessary (and is probably a waste of your time), but you can do it if you really want to. Otherwise, I think we should just wait for @urkud to review :)

Reply:
It's done. It was pretty easy, as there's really very few changes.
My mistake was keeping the file that I had worked with and modified many times instead of inserting my changes into a copy of the mathlib file.